### Exercício 10
**Nome**: Luiz Fernando Ferreira Berber <br>
**RA**: 817123421

**1.** Faça um Programa que peça dois números e imprima o maior deles.

In [2]:
num1 = int(input('Insira o primeiro número: '))
num2 = int(input('insira o segundo número: '))

if num1 > num2:
    print(num1)
else:
    print(num2)


Insira o primeiro número: 2
insira o segundo número: 1
2


**2.** Faça um Programa que verifique se uma letra digitada é vogal ou consoante.

In [5]:
vogal = ['a', 'e', 'i', 'o', 'u']

letra = str(input('Digite uma letra: '))

if letra in vogal:
    print('A letra é uma vogal!')
else:
    print('A leta é uma consoante!')

Digite uma letra: e
A letra é uma vogal!


**3.** Faça um programa para a leitura de duas notas parciais de um aluno. O programa
deve calcular a média alcançada por aluno e apresentar:
- A mensagem "Aprovado", se a média alcançada for maior ou igual a sete;
- A mensagem "Reprovado", se a média for menor do que sete;
- A mensagem "Aprovado com Distinção", se a média for igual a dez.

In [13]:
nota1 = int(input('Digite a nota 1: '))
nota2 = int(input('Digite a nota 2: '))

media = (nota1 + nota2) / 2

if media >= 7 and media <10:
    print('Aprovado!')
elif media < 7:
    print('Reprovado :(')
else:
    print('Aprovado com Distinção!')

Digite a nota 1: 10
Digite a nota 2: 10
Aprovado com Distinção!


**4.** Faça um Programa que leia três números e mostre-os em ordem decrescente

In [9]:
lista = [int(input('Digite o primeiro: ')), int(input('Digite o segundo: ')), int(input('Digite o segundo: '))]

lista.sort()

cont = len(lista)

while cont > 0:
    
    print(lista[cont-1])
    
    cont-= 1


Digite o primeiro: 10
Digite o segundo: 11
Digite o segundo: 15
15
11
10


**5.** As Organizações Tabajara resolveram dar um aumento de salário aos seus
colaboradores e lhe contrataram para desenvolver o programa que calculará os
reajustes.
Faça um programa que recebe o salário de um colaborador e o reajuste segundo o
seguinte critério, baseado no salário atual:
- salários até R$ 280,00 (incluindo) : aumento de 20%
- salários entre R$ 280,00 e R$ 700,00 : aumento de 15%
- salários entre R$ 700,00 e R$ 1500,00 : aumento de 10%
- salários de R$ 1500,00 em diante : aumento de 5% Após o aumento ser realizado,
informe na tela:
- o salário antes do reajuste;
- o percentual de aumento aplicado;
- o valor do aumento;
- o novo salário, após o aumento.

In [11]:
salario = int(input('Digite o salário: '))

aumento = 0
salario_total = 0
percentual = ""

if salario <= 280:
    aumento = salario*0.20
    salario_total = salario + aumento
    percentual = "20%"
elif salario > 280 and salario <= 700:
    aumento = salario*0.20
    salario_total = salario + aumento
    percentual = "20%"
elif salario > 700 and salario <= 1500:
    aumento = salario*0.10
    salario_total = salario + aumento
    percentual = "10%"
elif salario > 1500:
    aumento = salario*0.05
    salario_total = salario + aumento
    percentual = "5%"

print(f"Salário anterior: {salario}. Aumento: {aumento} Salário Atual: {salario_total} Aumento de: {percentual}")

Digite o salário: 600
Salário anterior: 600. Aumento: 120.0 Salário Atual: 720.0 Aumento de: 20%


**6.** Faça um Programa que leia um número e exiba o dia correspondente da semana.
(1-Domingo, 2- Segunda, etc.), se digitar outro valor deve aparecer valor inválido.

In [2]:
numero = int(input('Digite o número: '))

dia = ['Domingo', 'Segunda', 'Terça', 'Quarta', 'Quinta', 'Sexta', 'Sábado']

if numero <= 7 and numero >= 1:
    print(dia[numero - 1])
else:
    print('Número inválido')

Digite o número: 7
Sábado


**7.** Faça um programa que lê as duas notas parciais obtidas por um aluno numa
disciplina ao longo de um semestre, e calcule a sua média. A atribuição de conceitos
obedece à tabela abaixo:
 Média de Aproveitamento Conceito
 Entre 9.0 e 10.0 A
 Entre 7.5 e 9.0 B
 Entre 6.0 e 7.5 C
 Entre 4.0 e 6.0 D
 Entre 4.0 e zero E
O algoritmo deve mostrar na tela as notas, a média, o conceito correspondente e a
mensagem “APROVADO” se o conceito for A, B ou C ou “REPROVADO” se o conceito
for D ou E.

In [32]:
nota1 = int(input("Digite a primeira nota: "))
nota2 = int(input("Digite a segunda nota: "))

print( "A nota da primeira prova: ", nota1)
print( "A nota da segunda prova: ", nota2)

media = (nota1 + nota2)/2

if media >= 9 and media <= 10:
    print( "Você tirou um A!")
    print( "Sua media é de: ", media)
    print( "você foi aprovado!!")

elif media >= 7.5 and media < 9:
    print( "Você tirou um B!")
    print( "Sua media é de: ", media)
    print( "você foi aprovado!!")

elif media >= 6.5 and media < 7.5:
    print( "Você tirou um C!")
    print( "Sua media é de: ", media)
    print( "você foi aprovado!!")

elif media >= 4 and media < 6:
    print( "Você tirou um D!")
    print( "Sua media é de: ", media)
    print( "você foi reprovado!!")

elif media < 4:
    print( "Você tirou um E!")
    print( "Sua media é de: ", media)
    print( "você foi reprovado!!")

Digite a primeira nota: 10
Digite a segunda nota: 9
A nota da primeira prova:  10
A nota da segunda prova:  9
Você tirou um A!
Sua media é de:  9.5
você foi aprovado!!


**8.** Faça um Programa que peça os 3 lados de um triângulo. O programa deverá
informar se os valores podem ser um triângulo. Indique, caso os lados formem um
triângulo, se o mesmo é: equilátero, isósceles ou escaleno.
Dicas:
- Três lados formam um triângulo quando a soma de quaisquer dois lados for maior
que o terceiro;
- Triângulo Equilátero: três lados iguais;
- Triângulo Isósceles: quaisquer dois lados iguais;
- Triângulo Escaleno: três lados diferentes;

In [7]:
ladoA = int(input('insita o primeiro lado: '))
ladoB = int(input('insita o segundo lado: '))
ladoC = int(input('insita o terceiro lado: '))

if (ladoA + ladoB) > ladoC or (ladoA + ladoC) > ladoB or (ladoB + ladoC) > ladoA:
    print("Não pode ser um triangulo")
elif l1 == l2 or l1 == l3 or l2 == l3:
    print("Isósceles")
elif l1 == l2 == l3:
    print("Equilatero")
else:
    print("Escaleno")

insita o primeiro lado: 2
insita o segundo lado: 3
insita o terceiro lado: 8
Não pode ser um triangulo


**9.** Faça um programa que calcule as raízes de uma equação do segundo grau, na
forma ax2 + bx + c. O programa deverá pedir os valores de a, b e c e fazer as
consistências, informando ao usuário nas seguintes situações:
- Se o usuário informar o valor de A igual a zero, a equação não é do segundo grau e
o programa não deve fazer pedir os demais valores, sendo encerrado;
- Se o delta calculado for negativo, a equação não possui raizes reais. Informe ao
usuário e encerre o programa;
- Se o delta calculado for igual a zero a equação possui apenas uma raiz real;
informe-a ao usuário;
- Se o delta for positivo, a equação possui duas raiz reais; informe-as ao usuário;

In [ ]:
a = input('insira o valor de A: ')

if a != 0:
    b = input('insira o valor de B: ')
    x = input('insira o valor de X: ')
    c = input('insira o valor de C: ')
    
    delta = (a*x*2) + (b*x) + c
    
    if delta < 0:
        print('O delta {}, é negativo, a equalçao não possui raizes reais!'.format(delta))
    elif delta == 0:
        print('O delta, é igual a {}, logo, possui apenas uma raís'.format(delta))
    else:
    
elif:
    print('A equação não é de segundo grau, tchau!')



**10.** Faça um Programa para um caixa eletrônico. O programa deverá perguntar ao
usuário a valor do saque e depois informar quantas notas de cada valor serão
fornecidas. As notas disponíveis serão as de 1, 5, 10, 50 e 100 reais. O valor mínimo é
de 10 reais e o máximo de 600 reais. O programa não deve se preocupar com a
quantidade de notas existentes na máquina.
- Exemplo 1: Para sacar a quantia de 256 reais, o programa fornece duas notas de
100, uma nota de 50, uma nota de 5 e uma nota de 1;
- Exemplo 2: Para sacar a quantia de 399 reais, o programa fornece três notas de
100, uma nota de 50, quatro notas de 10, uma nota de 5 e quatro notas de 1.

In [35]:
notas = [1, 5, 10, 50, 100]
valor = int(input('Qual o valor? (validos entre 10 e 600): '))

if valor >= 10 and valor <= 600:
    quantia = 0
    saida = list()
    p = 4
    while quantia < valor and p >= 0:
        if valor - notas[p] >= 0:
            saida.append(notas[p])
            valor -= notas[p]
        else:
            p -= 1
    print('As notas são: ', saida)
else:
    print('Valor inválido.')

Qual o valor? (validos entre 10 e 600): 567
As notas são:  [100, 100, 100, 100, 100, 50, 10, 5, 1, 1]


**11.** Faça um programa que faça 5 perguntas para uma pessoa sobre um crime. As
perguntas são:
"Telefonou para a vítima?"
"Esteve no local do crime?"
"Mora perto da vítima?"
"Devia para a vítima?"
"Já trabalhou com a vítima?"
O programa deve no final emitir uma classificação sobre a participação da pessoa no
crime. Se a pessoa responder positivamente a 2 questões ela deve ser classificada
como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino". Caso contrário,
ele será classificado como "Inocente".

In [ ]:
interrogatorio = {"Telefonou para a vítima?":False,
    "Esteve no local do crime?":False, 
    "Mora perto da vítima?":False,
    "Devia para a vítima?":False,
    "Já trabalhou com a vítima?":False}
for item in interrogatorio:
    interrogatorio[item]=input(item + 'S/N :').upper()=='S'
respostas = (sum(interrogatorio.values()))

if respostas == 0 or respostas == 1:
    situacao = 'inocente'
elif respostas == 2:
    situacao = 'suspeito'
elif respostas == 3 or respostas == 4:
    situacao = 'cúmplice'
elif respostas == 5:
    situacao = 'assassino'
    
print(f'O investigado é {situacao}.')

**12.** Uma fruteira está vendendo frutas com a seguinte tabela de preços:
 Até 5 Kg Acima de 5 Kg
Morango R$ 2,50 por Kg R$ 2,20 por Kg
Maçã R$ 1,80 por Kg R$ 1,50 por Kg
Se o cliente comprar mais de 8 Kg em frutas ou o valor total da compra ultrapassar
R$ 25,00, receberá ainda um desconto de 10% sobre este total. Escreva um
algoritmo para ler a quantidade (em Kg) de morangos e a quantidade (em Kg) de
maças adquiridas e escreva o valor a ser pago pelo cliente.

In [37]:
morango = float(input('Digite quantos kilos de morango: '))
maça = float(input('Digite quantos kilos de maçâ: '))

valor_morango = 0
valor_maça = 0

if morango <= 5:
    valor_morango = morango*2.5
else:
    valor_morango = morango*2.2
    
if maça <= 5:
    valor_maça = maça*1.8
else:
    valor_maça = maça*1.5

valor_total = valor_maça + valor_morango

if valor_total >= 25 or morango + maça >= 8:
    valor_total = valor_total - (valor_total*0.1)

print('Valor total: {}'.format(valor_total))

Digite quantos kilos de morango: 10
Digite quantos kilos de maçâ: 8
Valor total: 30.6


**13.** Faça um programa que peça uma nota, entre zero e dez. Mostre uma mensagem
caso o valor seja inválido e continue pedindo até que o usuário informe um valor
válido.

In [48]:
while True:
    nota = int(input("Digite uma nota: "))
    
    if nota < 0 or nota > 10:
        print("Digite uma nota válida!")
    else:
        break
        

Digite uma nota: 11
Nota invalida
Digite uma nota: -12
Nota invalida
Digite uma nota: 5


**13.** Faça um programa que leia e valide as seguintes informações:
Nome: maior que 3 caracteres;
Idade: entre 0 e 150;
Salário: maior que zero;
Sexo: 'f' ou 'm';
Estado Civil: 's', 'c', 'v', 'd';

**14.** Faça um programa que leia 5 números e informe o maior número.

In [50]:
lista = [int(input('Digite o primeiro número: ')), int(input('Digite o segundo número: ')), 
         int(input('Digite o terceiro número: ')), int(input('Digite o quarto número: ')), 
         int(input('Digite o quinto número: '))]
            
lista.sort()

print('O maior número digitato foi: {}'.format(lista[len(lista)-1]))

Digite o primeiro número: 4
Digite o segundo número: 3
Digite o terceiro número: 7
Digite o quarto número: 3
Digite o quinto número: 9
O maior número digitato foi: 9


**15.** Faça um programa que leia 5 números e informe a soma e a média dos números.

In [53]:
lista = [int(input('Digite o primeiro número: ')), int(input('Digite o segundo número: ')), 
         int(input('Digite o terceiro número: ')), int(input('Digite o quarto número: ')), 
         int(input('Digite o quinto número: '))]

soma = 0
for i in lista:
    soma = soma + i;

print('Soma = {}. Média = {}'.format(soma, (soma / 5)))

Digite o primeiro número: 4
Digite o segundo número: 6
Digite o terceiro número: 5
Digite o quarto número: 7
Digite o quinto número: 3
Soma = 25. Média = 5.0


**16.** Faça um programa que imprima na tela apenas os números ímpares entre 1 e 50.

In [54]:
for i in range(1, 50):
    if  i%2 != 0:
        print(i)

1
3
5
7
9
11
13
15
17
19
21
23
25
27
29
31
33
35
37
39
41
43
45
47
49


**17.** Desenvolva um gerador de tabuada, capaz de gerar a tabuada de qualquer
número inteiro entre 1 a 10. O usuário deve informar de qual numero ele deseja ver
a tabuada. A saída deve ser conforme o exemplo abaixo:
Tabuada de 5:
5 X 1 = 5
5 X 2 = 10
...
5 X 10 = 50

In [66]:
num = int(input('Digite o numero da tabada: '))

if  num > 0 and num <= 10:
    
    for i in range(1, 10):
        print('{} X {} = {}'.format(num, i, (num*i)))



Digite o numero da tabada: 2
2 X 1 = 2
2 X 2 = 4
2 X 3 = 6
2 X 4 = 8
2 X 5 = 10
2 X 6 = 12
2 X 7 = 14
2 X 8 = 16
2 X 9 = 18


**18.** A série de Fibonacci é formada pela seqüência 1,1,2,3,5,8,13,21,34,55,... Faça um
programa capaz de gerar a série até o n−ésimo termo.

In [68]:
n = int(input('Digite o n-ésimo termo: '))

lista = [1,1]

for i in range(2,n):
    lista.append(lista[i-2]+lista[i-1])
if n == 1:
    print([1])
else:
    print(lista)

Digite o n-ésimo termo: 20
[1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597, 2584, 4181, 6765]


**19.** Faça um programa que calcule o fatorial de um número inteiro fornecido pelo
usuário. Ex.: 5!=5.4.3.2.1=120

In [70]:
num = int(input("Digite um numero: "))

fat = 1
x = 1

while x <= num:
    fat *= x
    x += 1
    
print(fat)

Digite um numero: 5
120


**20.** O Sr. Manoel Joaquim possui uma grande loja de artigos de R$ 1,99, com cerca
de 10 caixas. Para agilizar o cálculo de quanto cada cliente deve pagar ele
desenvolveu um tabela que contém o número de itens que o cliente comprou e ao
lado o valor da conta. Desta forma a atendente do caixa precisa apenas contar
quantos itens o cliente está levando e olhar na tabela de preços. Você foi contratado
para desenvolver o programa que monta esta tabela de preços, que conterá os
preços de 1 até 50 produtos, conforme o exemplo abaixo:
Lojas Quase Dois - Tabela de preços
1 - R$ 1.99
2 - R$ 3.98
...
50 - R$ 99.50

In [2]:
print('Lojas Quase Dois - Tabela de preços')
for i in range(1, 51):
    print('{} - R${}'.format(i, (i*1.99)))

Lojas Quase Dois - Tabela de preços
1 - R$1.99
2 - R$3.98
3 - R$5.97
4 - R$7.96
5 - R$9.95
6 - R$11.94
7 - R$13.93
8 - R$15.92
9 - R$17.91
10 - R$19.9
11 - R$21.89
12 - R$23.88
13 - R$25.87
14 - R$27.86
15 - R$29.85
16 - R$31.84
17 - R$33.83
18 - R$35.82
19 - R$37.81
20 - R$39.8
21 - R$41.79
22 - R$43.78
23 - R$45.77
24 - R$47.76
25 - R$49.75
26 - R$51.74
27 - R$53.73
28 - R$55.72
29 - R$57.71
30 - R$59.7
31 - R$61.69
32 - R$63.68
33 - R$65.67
34 - R$67.66
35 - R$69.65
36 - R$71.64
37 - R$73.63
38 - R$75.62
39 - R$77.61
40 - R$79.6
41 - R$81.59
42 - R$83.58
43 - R$85.57
44 - R$87.56
45 - R$89.55
46 - R$91.54
47 - R$93.53
48 - R$95.52
49 - R$97.51
50 - R$99.5


**21.** O Sr. Manoel Joaquim acaba de adquirir uma panificadora e pretende implantar
a metodologia da tabelinha, que já é um sucesso na sua loja de 1,99. Você foi
contratado para desenvolver o programa que monta a tabela de preços de pães, de
1 até 50 pães, a partir do preço do pão informado pelo usuário, conforme o exemplo
abaixo:
Preço do pão: R$ 0.18
Panificadora Pão de Ontem - Tabela de preços
1 - R$ 0.18
2 - R$ 0.36
...
50 - R$ 9.00

In [3]:
preco = float(input('Preço do pão: '))

print('Panificadora Pão de Ontem - Tabela de preços')
for i in range(1, 51):
    print('{} - R${}'.format(i, (i*preco)))

Preço do pão: 0.18
Panificadora Pão de Ontem - Tabela de preços
1 - R$0.18
2 - R$0.36
3 - R$0.54
4 - R$0.72
5 - R$0.8999999999999999
6 - R$1.08
7 - R$1.26
8 - R$1.44
9 - R$1.6199999999999999
10 - R$1.7999999999999998
11 - R$1.98
12 - R$2.16
13 - R$2.34
14 - R$2.52
15 - R$2.6999999999999997
16 - R$2.88
17 - R$3.06
18 - R$3.2399999999999998
19 - R$3.42
20 - R$3.5999999999999996
21 - R$3.78
22 - R$3.96
23 - R$4.14
24 - R$4.32
25 - R$4.5
26 - R$4.68
27 - R$4.859999999999999
28 - R$5.04
29 - R$5.22
30 - R$5.3999999999999995
31 - R$5.58
32 - R$5.76
33 - R$5.9399999999999995
34 - R$6.12
35 - R$6.3
36 - R$6.4799999999999995
37 - R$6.66
38 - R$6.84
39 - R$7.02
40 - R$7.199999999999999
41 - R$7.38
42 - R$7.56
43 - R$7.739999999999999
44 - R$7.92
45 - R$8.1
46 - R$8.28
47 - R$8.459999999999999
48 - R$8.64
49 - R$8.82
50 - R$9.0


**22.** O Sr. Manoel Joaquim expandiu seus negócios para além dos negócios de 1,99 e
agora possui uma loja de conveniências. Faça um programa que implemente uma
caixa registradora rudimentar. O programa deverá receber um número
desconhecido de valores referentes aos preços das mercadorias. Um valor zero deve
ser informado pelo operador para indicar o final da compra. O programa deve então
mostrar o total da compra e perguntar o valor em dinheiro que o cliente forneceu,
para então calcular e mostrar o valor do troco. Após esta operação, o programa
deverá voltar ao ponto inicial, para registrar a próxima compra. A saída deve ser
conforme o exemplo abaixo:
Lojas Tabajara
Produto 1: R$ 2.20
Produto 2: R$ 5.80
Produto 3: R$ 0
Total: R$ 9.00
Dinheiro: R$ 20.00
Troco: R$ 11.00

In [7]:
valores = []

print('Lojas Tabajara')

cont = 1

while True:
    valor = float(input(f'Produto {cont}: '))
    if(valor == 0):
        print(valores)
        break
    else:
        valores.append(valor)
    cont += 1
    


Lojas Tabajara
Produto : 11
Produto : 22
Produto : 30
[1.0, 2.0]


**23.** Foi feita uma estatística em cinco cidades brasileiras para coletar dados sobre
acidentes de trânsito. Foram obtidos os seguintes dados:
- Código da cidade;
- Número de veículos de passeio (em 1999);
- Número de acidentes de trânsito com vítimas (em 1999). Deseja-se saber:
- Qual o maior e menor índice de acidentes de transito e a que cidade pertence;
- Qual a média de veículos nas cinco cidades juntas;
- Qual a média de acidentes de trânsito nas cidades com menos de 2.000 veículos de
passeio.

In [7]:
cod_cidades = []
n_veiculos = []
n_acidentes = []

for i in range(5):
    cod_cidades.append(int(input('Entre com código cidade: ')))
    n_veiculos.append(int(input('Entre com a quantidade de veiculos : ')))
    n_acidentes.append(int(input('Entre com a quantidade de acidentes :')))

maior_indice_acidente = n_acidentes[0]
menor_indice_acidente = n_acidentes[0]
maior_index_acidente = 0
menor_index_acidentes = 0

for i, v in enumerate(n_acidentes):
    if maior_indice_acidente < v:
        maior_indice_acidente = v
        maior_index_acidente = i
    else:
        menor_indice_acidente = v
        menor_index_acidentes = i
        
media_veiculos = 0
soma_veiculos = 0 
cidades_mais_de_2k_veiculos = []
for i, v in enumerate(n_veiculos):
    soma_veiculos += v
    if v > 2000:
        cidades_mais_de_2k_veiculos.append(i)
        
media_veiculos = soma_veiculos/len(n_veiculos)

soma_acidentes = 0
media_acidentes = 0
for i, v in enumerate(n_acidentes):
    if not i in cidades_mais_de_2k_veiculos:
        soma_acidentes += v
        
media_acidentes = soma_acidentes/len(n_acidentes)

print('Maior indice de acidentes: ', maior_indice_acidente)
print('Menor indice de acidentes: ', menor_indice_acidente)
print('Cidade do maior indice de acidentes: ', cod_cidades[maior_index_acidente])
print('Cidade do menor indice de acidentes: ', cod_cidades[menor_index_acidentes])
print('Media de veiculos de todas as cidades: ', media_veiculos)
print('Media acidentes de transitos: ', media_acidentes)

Entre com código cidade: 10
Entre com a quantidade de veiculos : 8593
Entre com a quantidade de acidentes :453
Entre com código cidade: 11
Entre com a quantidade de veiculos : 4353
Entre com a quantidade de acidentes :123
Entre com código cidade: 12
Entre com a quantidade de veiculos : 8786
Entre com a quantidade de acidentes :232
Entre com código cidade: 13
Entre com a quantidade de veiculos : 465654
Entre com a quantidade de acidentes :343
Entre com código cidade: 14
Entre com a quantidade de veiculos : 8695
Entre com a quantidade de acidentes :34
Maior indice de acidentes:  453
Menor indice de acidentes:  34
Cidade do maior indice de acidentes:  10
Cidade do menor indice de acidentes:  14
Media de veiculos de todas as cidades:  99216.2
Media acidentes de transitos:  0.0


**24.** Faça um programa que receba o valor de uma dívida e mostre uma tabela com
os seguintes dados: valor da dívida, valor dos juros, quantidade de parcelas e valor
da parcela.
Os juros e a quantidade de parcelas seguem a tabela abaixo:
Quantidade de Parcelas % de Juros sobre o valor inicial da dívida
1 0
3 10
6 15
9 20
12 25
Exemplo de saída do programa:
Valor da Dívida Valor dos Juros Quantidade de Parcelas Valor da Parcela
R$ 1.000,00 0 1 R$ 1.000,00
R$ 1.100,00 100 3 R$ 366,00
R$ 1.150,00 150 6 R$ 191,67

In [4]:
divida = float(input("Digite valor da divida"))
juros = 0
parcelas = 3
for x in range(5):
  print('Valor da Divida',divida,
  'valor do juros ', juros*(divida/100),
  'quantidade de apercelas', parcelas * (x),
  ' valor da parcela ', divida+(juros*(divida/100))
  )
  juros +=5

Digite valor da divida1000
Valor da Divida 1000.0 valor do juros  0.0 quantidade de apercelas 0  valor da parcela  1000.0
Valor da Divida 1000.0 valor do juros  50.0 quantidade de apercelas 3  valor da parcela  1050.0
Valor da Divida 1000.0 valor do juros  100.0 quantidade de apercelas 6  valor da parcela  1100.0
Valor da Divida 1000.0 valor do juros  150.0 quantidade de apercelas 9  valor da parcela  1150.0
Valor da Divida 1000.0 valor do juros  200.0 quantidade de apercelas 12  valor da parcela  1200.0


**25.** Faça um programa que mostre os n termos da Série a seguir:
 S = 1/1 + 2/3 + 3/5 + 4/7 + 5/9 + ... + n/m.
Imprima no final a soma da série.

In [11]:
n = int(input('entre com o n: '))
m = int(input('entre com o m: '))

def mostra(n, m):
    m = (m/2)
    result = ''
    i = 1
    s = 1
    while(i <= m):
        s+=2
        result += str(i+1)+'/'+str(s)+' + '
        i+=1
    return result[0:len(result)-3]
    
def soma(n, m):
    m = (m/2)
    soma = 0
    i = 1
    while(i <= m):
        soma += ((i+1)/(i+2))
        i+=1
    soma += (1/1)
    return soma    
    
if n >= 1 and m >= 1:
    soma = soma(n, m)
    print('S = 1/1 + '+ mostra(n, m))
    print('Soma: %.2f' %soma)
else: 
    print('Números menores que 1')

entre com o n: 2
entre com o m: 3
S = 1/1 + 2/3
Soma: 1.67
